src/components/rpc_base/test/CMakeLists.txt

```diff
@@ -31,9 +31,8 @@
 if (BUILD_TESTS)
```

Comment:
@LevchenkoS , there is fixed. But in another places will be fixed in planed task.